js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useStaticSetFilter.tsx

```diff
@@ -94,27 +100,69 @@ export function useStaticSetFilter<TValue>({
             key: getKey?.(value) || index.toString(),
             value,
           }));
+        } else {
+          results = allValues
+            .filter(({match}) =>
+              match.some((value) => value.toLowerCase().includes(query.toLowerCase())),
+            )
+            .map(({value}, index) => ({
+              label: (
+                <SetFilterLabel
+                  value={value}
+                  renderLabel={renderLabel}
+                  filter={filterObjRef.current}
+                  allowMultipleSelections={allowMultipleSelections}
+                />
+              ),
+              key: getKey?.(value) || index.toString(),
+              value,
+            }));
         }
-
-        return allValues
-          .filter(({match}) =>
-            match.some((value) => value.toLowerCase().includes(query.toLowerCase())),
-          )
-          .map(({value}, index) => ({
-            label: (
-              <SetFilterLabel
-                value={value}
-                renderLabel={renderLabel}
-                filter={filterObjRef.current}
-                allowMultipleSelections={allowMultipleSelections}
-              />
-            ),
-            key: getKey?.(value) || index.toString(),
-            value,
-          }));
+        if (allowMultipleSelections) {
+          return [
+            {
+              label: <ButtonLink>Select all</ButtonLink>,
+              key: 'useStaticSetFilter:select-all-key',
+              value: selectAllSymbol,
+            },
```

Comment:
Can I alternatively unselect all?

Reply:
Yeah that's what it does

Reply:
I was considering changing this to "Toggle all", but figured users would probably just click it again and see it does toggle all